neondatabase/website
# feat: add Changelog page

**Describe** what changes this pull request brings
This pull request adds Changelog page.

**Steps to test**
1. Open the [page](https://websitemain62807-changelog.gtsb.io/docs/release-notes/)
2. Confirm that everything looks and works as expected

<details>
<summary>Screenshots</summary>

 Changelog page
<img width="2043" alt="Screenshot 2022-08-01 at 17 45 49" src="https://user-images.githubusercontent.com/75138180/182152384-009a55d6-e4a6-48aa-b460-c3c4c7695433.png">

 Changelog page pagination
<img width="2045" alt="Screenshot 2022-08-01 at 17 46 22" src="https://user-images.githubusercontent.com/75138180/182152326-0490c2ef-11a7-4b3a-8047-12af1f1784ac.png">


 Changelog post
<img width="2045" alt="Screenshot 2022-08-01 at 17 46 29" src="https://user-images.githubusercontent.com/75138180/182152209-5d787e2c-8fa4-421e-afa5-fadc3034e20c.png">

</details>

**References**
[Design](https://www.figma.com/file/6WyZRL33uvZsBewEWxB3Si/Neon-Website-Design?node-id=6108%3A15447)
